.gitignore

```diff
@@ -1,2 +1,4 @@
+Gemfile.lock
```

Comment:
`Gemfile.lock` should actually be included to make sure that everyone is using the same library versions.